sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -241,10 +241,98 @@ def from_connection_string(
         )
         return cls(**constructor_args)
 
+    async def _receive(
+            self,
+            on_event,
+            batch=False,
+            *,
+            max_batch_size: int = 300,
+            max_wait_time: Optional[float] = 3,
```

Comment:
Why did we decide to default this to 3? Is that based on some team concensus?

Reply:
This is a mistake. Should be None instead of 3.